### ISSUE TITLE
Disabling getVideoTags not working properly

### ISSUE THREAD

**Issue by akhilleusuggo** [WWBN/AVideo#4359]
Even disabling getVideoTags , still is trying to pull tags from HLS videos only ( index.m3u8 ) , working properly for MP4

**Comment by DanielnetoDotCom** [comment]
what tags exactly are you referring to?

please screenshot

**Comment by akhilleusuggo** [comment]
@DanielnetoDotCom  CustomizeAdvanced
![image](https://user-images.githubusercontent.com/52116577/104738554-57616b80-574e-11eb-9faf-14fff02fa5d7.png)

**Comment by DanielnetoDotCom** [comment]
I found something and already sent the update.

the HLS format when you are using the Storage was slowing down when tries to get the resolutions.
It was waiting, sometimes, 10 seconds to wait for the storage response. 

VideoHLS v4.1 will only wait 2 seconds, if the storage does not respond it will not get the resolution.
also on this update, it does not handshake (SSL) the storage anymore

Something that improves my side was I added on the hosts file the internal IP for the storage server,  so it does not transit in the public network anymore.

try to update the streamer and plugin ant tell me the results
